src/main/java/seedu/address/model/AddressBook.java

```diff
@@ -106,7 +106,8 @@ public void removeStudent(Student key) {
     public void addSession(Name name, Session session) {
         requireAllNonNull(session, name);
         Student student = students.getStudentWithName(name);
-        student.addSession(session);
+        // student.addSession(session);
```

Comment:
think you forgot to remove the comment right here

Reply:
ohmilorde thanku